src/azure-cli/azure/cli/command_modules/rdbms/commands.py

```diff
@@ -439,16 +439,16 @@ def load_command_table(self, _):
                             mysql_adadmin_sdk,
                             client_factory=cf_mysql_server_ad_administrators_operations,
                             is_preview=True) as g:
-        g.custom_command('create', 'server_ad_admin_set')
+        g.custom_command('create', 'server_ad_admin_set', supports_no_wait=True)
         g.command('list', 'list')
-        g.command('delete', 'delete')
+        g.command('delete', 'delete', confirmation=True)
         g.command('show', 'get')
```

Comment:
All show commands should be registered using the show_command or custom_show_command helpers to ensure 404(Not Found) is always returning an exit code of 3.
https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md#standard-command-types